torch/_dynamo/variables/builder.py

```diff
@@ -570,14 +571,20 @@ def build_key_value(k, v):
         elif isinstance(value, HigherOrderOperator):
             self.install_guards(GuardBuilder.TYPE_MATCH, GuardBuilder.NAME_MATCH)
             return TorchHigherOrderOperatorVariable.make(value, source=self.source)
+        elif isinstance(value, torch.cuda.StreamContext):
+            self.install_guards(GuardBuilder.ID_MATCH)
+            stream_source = AttrSource(self.source, "stream")
+            stream_var = VariableBuilder(self.tx, stream_source)(value.stream)
+            return StreamContextVariable.create_from_stream(self.tx, stream_var)
```

Comment:
This is apparently preexisting, so no action needed, but it's weird to me that we need a dedicated StreamContextVariable, it seems like a pretty standard context manager around stream setting apis, it seems to me that we should just inline into it.